docs/docs/configuration/enum.mdx

```diff
@@ -182,18 +182,18 @@ public partial class CarMapper
 
 By default Mapperly emits diagnostics with a severity of warning if there are unmapped source or target enum members.
 To enforce strict enum member mappings on only either the source or the target,
-the `RequiredMappingStrategy` can be used.
+the `RequiredEnumMappingStrategy` can be used. By default this property is set to `Inherit`, which will inherit the value from `RequiredMappingStrategy`.
```

Comment:
I think the `By default ...` sentence needs to be updated as it inherits but it is not set to a member Inherit...

Reply:
Fixed.